docker/development/debian/Dockerfile

```diff
@@ -64,6 +64,20 @@ RUN mkdir -p /tmp/open-space-toolkit-core \
  && apt-get install -y ./*.deb \
  && rm -rf /tmp/open-space-toolkit-core
 
+## Boost
+
+ARG BOOST_MAJOR_VERSION="1"
+ARG BOOST_MINOR_VERSION="82"
+ARG BOOST_VERSION="${BOOST_MAJOR_VERSION}.${BOOST_MINOR_VERSION}.0"
+
+RUN mkdir -p /tmp/boost \
+ && cd /tmp/boost \
+ && wget --quiet -O boost_${BOOST_VERSION}.tar.gz https://sourceforge.net/projects/boost/files/boost/${BOOST_VERSION}/boost_${BOOST_MAJOR_VERSION}_${BOOST_MINOR_VERSION}_0.tar.gz/download \
+ && tar xzvf boost_${BOOST_VERSION}.tar.gz \
+ && cd boost_${BOOST_MAJOR_VERSION}_${BOOST_MINOR_VERSION}_0 \
+ && ./bootstrap.sh --prefix=/usr/local \
+ && ./b2 install
```

Comment:
Over-installing Boost here is risky, as it may (likely will) trigger incompatibilities with other OSTk libs that link against another version. I recommend updating the base image here: https://github.com/open-space-collective/open-space-toolkit/blob/main/docker/development/debian/Dockerfile

Reply:
Ah that's a great idea. Will do.